docs/source/using/index.adoc

```diff
@@ -1,14 +1,12 @@
-[[using-index]]
+[[using-minishift]]
 = Using Minishift
```

Comment:
Let's stick with "Using" for now. You are managing a server, but using s CLI. That's at least where I am coming from atm.